spec/services/api/participant_outcomes/index_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Api::ParticipantOutcomes::Index, :with_default_schedules do
+  describe "#scope" do
+    let(:provider_declaration) { create_declaration }
+    let!(:provider_outcomes) { create_outcomes(provider_declaration) }
+    let(:other_provider_declaration) { create_declaration }
+    let!(:other_provider_outcomes) { create_outcomes(other_provider_declaration) }
+    subject(:index) { described_class.new(cpd_lead_provider: provider_declaration.cpd_lead_provider) }
+
+    it "returns all declarations outcomes for the provider" do
+      expect(index.scope.to_a).to eq(provider_outcomes)
+    end
+
+    it "returns outcomes across multiple declarations" do
```

Comment:
I wonder if we can instead of reaching into declarations and updating the provider, we can create provider 1 and provider 2 outside the create_declaration, and just pass them into the declaration as need be depending on the context? Also maybe we can set those declarations to completed to ensure the tests represent that they are only for completed declarations?

Reply:
I don't know what it would gain apart from making the test more complex.

Reply:
Let's catchup tomorrow about this, will be easier to show you what I have locally

Reply:
caught up in person but for posterity: there are different takes on how to go about this, a discussion for the wider team. ok to leave as is for now